app/master/cluster_master.py

```diff
@@ -296,7 +297,7 @@ def handle_result_reported_from_slave(self, slave_url, build_id, subjob_id, payl
         :rtype: str
         """
         self._logger.info('Results received from {} for subjob. (Build {}, Subjob {})', slave_url, build_id, subjob_id)
-        build = BuildStore.get(int(build_id))
+        build = self._build_store.get(int(build_id))
```

Comment:
Not your code but why is this converted to an `int`. Is the type in the docstring wrong?